build/lib/csiplugin-connector.service

```diff
@@ -3,11 +3,9 @@ Description=csiplugin connector
 After=network.target remote-fs.target nss-lookup.target
 
 [Service]
-Type=forking
-RuntimeDirectory=csiplugin
+Type=simple
+RuntimeDirectory=csi-tool/connector
```

Comment:
Two level of directory is required, because the inner one will be removed by systemd when the service is stopped/restarted.